src/Controllers/MovieController.cs

```diff
@@ -67,11 +67,29 @@ public IActionResult EditMovie(int id)
             }
         }
 
+        [HttpGet("")]
         [HttpGet("ListMovies")]
         public IActionResult ListMovies(){
             IEnumerable movieList = UnitOfWork.Movies.GetAll();
             return View(movieList);
         }
+
+        [HttpGet("DeleteMovie/{id}")]
+        public IActionResult DeleteMovie(int id){
+            //el try catch está por las dudas porque no sé lo que pasa cuando trato
```

Comment:
Great job! Try-catch is what we intend you to use in case that happens. When the app tries to delete a movie that not exists, it should throws an exception, but the way you are doing it here allows you to customize the message and get  more descriptive info on the error

Reply:
BTW, don't forget to remove this comment